person.rb

```diff
@@ -1,5 +1,10 @@
-class Person
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
```

Comment:
- [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.